src/css/components/_card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+    width: 50vw;
+    height: 20vh;
+    margin: 1rem;
+    background-color: orangered;
+
+    font-size: $font-size-interactive-large;
+    padding: 1rem;
+    display: grid;
+    grid-template-columns: [col-1-start] 70% [col-1-end col-2-start] 30%;
```

Comment:
I'm not sure if this should be abstracted out as a layout class. More research needs to be done on sass and proper 7-1 architecture